plugins/storage/volume/scaleio/src/main/java/org/apache/cloudstack/storage/datastore/driver/ScaleIOPrimaryDataStoreDriver.java

```diff
@@ -629,7 +629,8 @@ public void copyAsync(DataObject srcData, DataObject destData, Host destHost, As
 
     private Answer copyTemplateToVolume(DataObject srcData, DataObject destData, Host destHost) {
         // Copy PowerFlex/ScaleIO template to volume
-        LOGGER.debug("Initiating copy from PowerFlex template volume on host " + destHost != null ? destHost.getId() : "");
+        String debugMessage = "Initiating copy from PowerFlex template volume on host ";
+        LOGGER.debug(destHost != null ? debugMessage + destHost.getId() : debugMessage);
```

Comment:
We could use `String.format` here:

```suggestion
        LOGGER.debug(String.format("Initiating copy from PowerFlex template volume on host %s", destHost != null ? destHost.getId() : ""));
        
```

Reply:
Good call!